.gitattributes

```diff
@@ -1,2 +1,16 @@
-tests/nnapi/specs/* linguist-detectable=false
-res/* linguist-detectable=false
+# Exclude from git language statistics
+tests/nnapi/specs/** linguist-detectable=false
+res/** linguist-detectable=false
+
+# Default: text file
+# - Set End-Of-File type
```

Comment:
Q) this says `End-Of-File` but below attribute is `eol` which I understand is `End-Of-Line`. is this typo or EOL also applies to EOF?

Reply:
It's typo. :) Thanks.